fix(curriculum): Split Cat App Step 12 into two parts (#47320)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

# --description--

You can see the words `cat photos` underlined next to the image in the preview now. That's your link; feel free to give it a click.

> In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com` by adding a new anchor element.

Author: ```suggestion
In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com` by nesting these words inside of a new anchor element.
```

# --hints--

> Your anchor (`a`) element should be nested within the `p` element.

Reviewer: ```suggestion
You should nest a new anchor (`a`) element within the `p` element.
```

Considering that the step has changed, it makes sense to adapt the tests. Otherwise we risk to not alleviate the confusion.
